source/command-monitoring/command-monitoring.rst

```diff
@@ -179,25 +123,10 @@ A document sequence in an OP_MSG reply to "aggregate" or "getMore" MUST be simil
 
 See "Why are document sequences included as BSON arrays?" in the `rationale`_.
 
-Read Preference
-^^^^^^^^^^^^^^^
-
-In cases where queries or commands are embedded in a ``$query`` parameter when a read preference
```

Comment:
@jmikola and I discussed this section [here](https://github.com/mongodb/specifications/pull/1221#discussion_r879756606) back when converting the tests to the unified format. The section is a bit confusing but I think it was referring to OP_QUERY behavior as described [here](https://github.com/mongodb/specifications/blob/master/source/find_getmore_killcursors_commands.rst#readpreference-and-mongos) so it does not seem relevant for OP_MSG.